src/pytorch_lightning/loggers/tensorboard.py

```diff
@@ -188,6 +197,12 @@ def experiment(self) -> SummaryWriter:
         assert rank_zero_only.rank == 0, "tried to init log dirs in non global_rank=0"
         if self.root_dir:
             self._fs.makedirs(self.root_dir, exist_ok=True)
+
+        if _TENSORBOARD_AVAILABLE:
+            from torch.utils.tensorboard import SummaryWriter
+        else:
+            from tensorboardX import SummaryWriter  # type: ignore[no-redef]
```

Comment:
how many times is this called per epoch?

Reply:
Just once. Importing it again is a no-op

Reply:
@Borda Here's a quick example of importing the same thing multiple times: https://stackoverflow.com/questions/37067414/python-import-multiple-times